### DISCUSSION TITLE
What is the procedure for obtaining contributions that encompass additional parameters such as `extra_features`?

### DISCUSSION THREAD

**Discussion by cureyumyum** [google/lightweight_mmm#152] [Q&A]

**Comment by statm3n** [comment]
Right now contributions of extra_features are included in the baseline https://github.com/google/lightweight_mmm/blob/8152c43c8b52eb8c020c1c54c7e4bfa5140e42d0/lightweight_mmm/plot.py#L1011
 You can find the calculation here: https://github.com/google/lightweight_mmm/blob/8152c43c8b52eb8c020c1c54c7e4bfa5140e42d0/lightweight_mmm/plot.py#L251C43-L251C43

**Reply by DongHarry-Kang** [reply]
@statm3n I am new learner of data analytics; could you write an one-liner, or how to use above so that I can get extra features? Thank you very much!
